openedx/core/djangoapps/user_api/partition_schemes.py

```diff
@@ -77,3 +78,15 @@ def _key_for_partition(cls, user_partition):
         Returns the key to use to look up and save the user's group for a given user partition.
         """
         return 'xblock.partition_service.partition_{0}'.format(user_partition.id)
+
+
+def get_random_schemed_user_partitions(course_key):
```

Comment:
Instead of introducing this method, how about using the existing get_split_user_partitions method in split_test_module.py?